### PR TITLE
update money version

### PR DESCRIPTION
**Pull request by Karlotcha**

### CODE REVIEW

**Review by smnguyen — APPROVED** [review]
Seems legit 🤞

**Comment by zp-alfred** [comment]
Accepted by smnguyen!

**Comment by smnguyen** [comment]
Actually I think we might run into the same errors as my PR to upgrade money-rails altogether.

These two things in the changelog for money look like the errors I saw:
> Money#== will now raise error for non-zero numeric values
> Changed Money#<=> to return nil if the comparison is inappropriate. (#584)

I'm going to try running some of those tests locally with this upgrade

**Comment by smnguyen** [comment]
@Karlotcha Running `bundle install` fails actually, so I think we'll need to do a bit more to upgrade 😭

File: money-rails.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
 
   s.require_path = "lib"
 
-  s.add_dependency "money",         "~> 6.5.0"
+  s.add_dependency "money",         "~> 6.8.2"
   s.add_dependency "monetize",      "~> 1.1.0"
```
Comment (smnguyen) [inline money-rails.gemspec:30]: I got the build working by bumping this to 1.6.0. I can't run tests though, on this branch or master 😕

**Comment by smnguyen** [comment]
I pointed my local ZP to use this branch of `money-rails` and tried running some of the tests that failed when I upgraded `money-rails`. The tests failed with the same errors, so I think we'd have to make some actual code changes in ZP to be able to upgrade `money`.

**Pull request closed without merging**